src/app/Components/adminpanel/user-management/user-management.component.html

```diff
@@ -10,39 +10,34 @@
       </mat-expansion-panel-header>
 
       <div class="example-card example-half-width">
-        
         <hr class="example-full-width">
-        
-        <mat-card-content>
+        <mat-card-content fxLayout="column" fxLayoutAlign="space-around center">
           <mat-form-field class="example-full-width">
-            <input type="text" [(ngModel)]="nameSearch" (keyup)="applyFilter($event.target.value)" autocomplete="off"
-            matInput placeholder="User's name">
+            <input type="text" [(ngModel)]="searchInput" autocomplete="off" matInput placeholder="User's name">
           </mat-form-field>
           <br>
           <mat-form-field class="example-full-width">
             <mat-select [(value)]="selectedStatus" placeholder="Status" [(value)]="selected">
               <mat-option>None</mat-option>
-              <mat-option *ngFor="let item of Types" [value]="item.statusName">
-                {{item.statusName}}
+              <mat-option *ngFor="let item of statuses" [value]="item.id">
+                {{item.name}}
               </mat-option>
             </mat-select>
           </mat-form-field>
           <br>
-          <button (click)="onSearch()" class="example-full-width" mat-raised-button color="primary">Search</button>
+          <div id="button-box" fxLayout="row" fxLayoutAlign="space-around center">
+            <button (click)="onSearch()" class="example-full-width" mat-raised-button color="primary">Search</button>
```

Comment:
Why name of class starts from "example"?

Reply:
removed